dimager/templates/registration/activate.html

```diff
@@ -0,0 +1,16 @@
+{% extends "base.html" %}
+{% load i18n %}
+
+{% block content %}
+<p>{% trans "Account activation failed." %}</p>
+{% endblock %}
+
+
+{% comment %}
+**registration/activate.html**
+
+Used if account activation fails. With the default setup, has the following context:
+
+``activation_key``
+    The activation key used during the activation attempt.
+{% endcomment %}
```

Comment:
These templates are clearly copied from elsewhere.  But I don't see anything in your README that acknowledges the debt.  Please make sure you credit your sources.  Also, it's a waste to leave all those comments in place.  You don't need 'em.